types/block.go

```diff
@@ -0,0 +1,44 @@
+package types
+
+import (
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/streamingfast/bstream"
+	pbnear "github.com/streamingfast/firehose-near/types/pb/sf/near/type/v1"
+	pbbstream "github.com/streamingfast/pbgo/sf/bstream/v1"
+)
+
+func BlockFromProtoCodec(b *pbnear.Block) (*bstream.Block, error) {
```

Comment:
Hmmm why do we have two version? There are not aligned and I've seen usage of both in the PR, need to keep only one (if unsure check on `develop` branch which one is the correct one).

Remove the other one and renamed back to `BlockFromProto`